r2/r2/lib/jsontemplates.py

```diff
@@ -750,6 +750,9 @@ def generate_image_links(preview_object, censor_nsfw=False):
     def raw_data(self, thing):
         d = ThingJsonTemplate.raw_data(self, thing)
 
+        if c.user.gold and thing.saved:
+            d["saved_category"] = thing.saved_category if thing.saved_category else None
```

Comment:
This if expression seemed nicer, since "no category" would be made None, instead of empty quotes.